app/src/main/java/org/dhis2/usescases/splash/SplashActivity.kt

```diff
@@ -62,7 +64,14 @@ class SplashActivity : ActivityGlobalAbstract(), SplashView {
         super.onResume()
 
         if (BuildConfig.DEBUG || !RootBeer(this).isRootedWithoutBusyBoxCheck) {
-            presenter.init()
+            if (!isDebuggerEnable() || !detectDebugger()) {
```

Comment:
When we have BuildConfig.DEBUG, shoudn't we go to the init anyway?

Reply:
Both methods take in consideration if we are in debug.